### ISSUE TITLE
Can't download at all

### ISSUE THREAD

**Issue by rokonio** [AllToMP3/alltomp3-app#198]
* My operating system is: Windows 10.
* My AllToMP3 version is: I don't find were is it. But I installed it today.
* I just typed "stressed out" and select the first song

The download does not run. It's blocked. I stoped the firewall and delete the roaming/alltomp3 file and reinstall the app but it's still blocked. On the latest issues I don't see answer from the author so Is he still active ? I just want a anwer from him.

Sorry for the bad English I don't speak very well.
Thanks.

**Comment by marmundo** [comment]
Same thing here

**Comment by Codingboy65** [comment]
Yeah since the system changes often it's hard to update the app to catch up

**Comment by inglesuniversal** [comment]
no link works... so sad..... mac os big sur beta

**Comment by osman144** [comment]
Same nothing working at the moment on my side 

I'm on Mac os Catalina

**Comment by itzTheMeow** [comment]
yt-dl is what this uses I think to get songs from youtube, and the yt-dl repository has a copyright strike, so that *could* be the issue?

**Comment by inglesuniversal** [comment]
Any new release I can try?

**Comment by Sanches917** [comment]
me too windows 1909

**Comment by inglesuniversal** [comment]
confirmed!.... code is outdated!

**Comment by Andrea5883** [comment]
Same here, both YouTube or Spotify won't download at all.

**Comment by mariavalso** [comment]
hola 
no puedo descargar nada por favor me podeis decir una solucion 
gracias

**Comment by Sanches917** [comment]
This tool dead

**Comment by vernejules555** [comment]
Does this mean it won't work anymore?

**Comment by NTag** [comment]
I released an update. However it only supports YouTube URLs, Spotify/Deezer/song title searches are broken for now, sorry.

**Comment by itzTheMeow** [comment]
ok, sounds good, looking forward to the spotify URLs :3

**Comment by rokonio** [comment]
I encourage you. It's always better than nothing
